src/routers/features/cdc/bonusRequest.ts

```diff
@@ -0,0 +1,116 @@
+import { Router } from "express";
+import * as E from "fp-ts/lib/Either";
+import { AnniRiferimento } from "../../../../generated/definitions/cdc/AnniRiferimento";
+import { Anno } from "../../../../generated/definitions/cdc/Anno";
+import { EsitoRichiestaEnum } from "../../../../generated/definitions/cdc/EsitoRichiesta";
+import { ListaEsitoRichiestaPerAnno } from "../../../../generated/definitions/cdc/ListaEsitoRichiestaPerAnno";
+import { ListaStatoPerAnno } from "../../../../generated/definitions/cdc/ListaStatoPerAnno";
+import { StatoBeneficiarioEnum } from "../../../../generated/definitions/cdc/StatoBeneficiario";
+import { addHandler } from "../../../payloads/response";
+
+export const cdcBonusRequestRouter = Router();
+
+export const generateBonusAll = (): ListaStatoPerAnno => {
+  return {
+    listaStatoPerAnno: [
+      {
+        annoRiferimento: "2020" as Anno,
+        statoBeneficiario: StatoBeneficiarioEnum.ATTVABILE
+      },
+      {
+        annoRiferimento: "2021" as Anno,
+        statoBeneficiario: StatoBeneficiarioEnum.ATTVABILE
+      },
+      {
+        annoRiferimento: "2022" as Anno,
+        statoBeneficiario: StatoBeneficiarioEnum.ATTVABILE
+      }
+    ]
+  };
+};
+
+// TODO: update the prefix when will be official
+const addPrefix = (path: string) => `/bonus${path}`;
+
+// tslint:disable-next-line: no-let
+export let bonusAll: ListaStatoPerAnno = generateBonusAll();
+
+addHandler(
+  cdcBonusRequestRouter,
+  "get",
+  addPrefix("/beneficiario/stato"),
+  (req, res) => {
+    return res.status(200).json(bonusAll);
+  }
```

Comment:
```suggestion
  (req, res) => res.json(bonusAll)
```
`res.json` is already a 200 response

Reply:
Thanks, I didn't know. Removed in [94b1bb8](https://github.com/pagopa/io-dev-api-server/pull/167/commits/94b1bb8b715263b90753f398fd0700b12a9639ef)